Number of Islands.py

```diff
@@ -0,0 +1,24 @@
+'''
+TC: O(n)
+SC: O(n)
+'''
```

Comment:
mention what your N stands for 

TIme complexity would be O(MxN)
and space (1) since you are not using any extra space.